### DISCUSSION TITLE
Turn "Equipped" invsee GUI into a hopper instead of a chest

### DISCUSSION THREAD

**Discussion by ghost** [EssentialsX/Essentials#3972] [Ideas and Feature Suggestions] [answered]
This would be visually better since it has the exact amount of needed slots. The first 4 slots can be used for the player armor, and the fifth slot for the offhand item.
Also a way to modify others "Equipped" slots would be really useful.

**Comment by pop4959** [comment, marked as answer]
Slots for an inventory have to be a multiple of 9 which is why it's displayed the way it is currently. Using a hopper inventory would technically work, but I tested this and it displays the entire user inventory alongside it, which is not ideal and actually looks worse than just leaving it the way it is currently. As for modifying equipped slots, it's not possible due to the way that invsee works currently. This is also why invsee equip is considered an easter egg and not a proper feature.
